huspacy/tools/components.py

```diff
@@ -69,6 +70,61 @@ def initialize(
     pass
 
 
+class LookupLemmatizer(Pipe):
+    # noinspection PyUnusedLocal
+    @staticmethod
+    @Hungarian.factory(
+        "lookup_lemmatizer",
+        assigns=["token.lemma"],
+        requires=["token.pos"],
+        default_config={
+            "scorer": {"@scorers": "spacy.lemmatizer_scorer.v1"},
+            "source": ""
+        },
+    )
+    def create(nlp: Language, name: str, scorer: Optional[Callable], source: str) -> "LookupLemmatizer":
+        return LookupLemmatizer(None, source, scorer)
+
+    def __init__(self, lookups: Optional[Lookups], source: str, scorer: Optional[Callable] = lemmatizer_score):
+        self._lookups: Optional[Lookups] = lookups
+        self.scorer = scorer
+        self.source = source
+
+    def __call__(self, doc: Doc) -> Doc:
+        assert self._lookups is not None, "Lookup table should be initialized first"
+
+        token: Token
+        for token in doc:
+            if token.text in self._lookups.get_table(f"lemma_lookups_{token.tag_}"):
```

Comment:
Just a minor thing: this is very readable, but not very memory efficient. I might consider using a shorter table name. 

Is there a naming convention in spacy for that?

Reply:
In the Polish 3.2.x model they're using this naming convention. In theory, this should allow us to use the official Lemmatizer pipe, but I cannot see how.

Reply:
Thanks for the info. Than let's stick w/ the naming conventions.